scene/gui/rich_text_label.cpp

```diff
@@ -1881,7 +1910,10 @@ void RichTextLabel::_bind_methods() {
 	ADD_GROUP("BBCode", "bbcode_");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "bbcode_enabled"), "set_use_bbcode", "is_using_bbcode");
 	ADD_PROPERTY(PropertyInfo(Variant::STRING, "bbcode_text", PROPERTY_HINT_MULTILINE_TEXT), "set_bbcode", "get_bbcode");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "visible_characters", PROPERTY_HINT_RANGE, "-1,128000,1"), "set_visible_characters", "get_visible_characters");
+
+	ADD_GROUP("Character Visibility", "vis_");
```

Comment:
That group prefix doesn't feel very "Godot-like", as we usually try to avoid abbreviations.

Given that RTL does not have many properties, you could maybe remove the group altogether, as in Label.